modules/schemas/src/main/resources/contexts/nexus/core/links/v0.2.0.json

```diff
@@ -0,0 +1,35 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
```

Comment:
The following context {{base}}/contexts/nexus/core/standards/v0.1.0 needs to reuse the resource one. Is it the case ?

Reply:
I guess it will be added during run time.